frontend/src/features/projects/spl/steps/GreTransferStep.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import { Container, Form } from 'react-bootstrap';
+
+import {
+  IStepProps,
+  useStepForm,
+  IProject,
+  StepStatusIcon,
+  ReviewWorkflowStatus,
+  updatePimsWarning,
+  useProject,
+} from '../../common';
+
+import { Formik } from 'formik';
+import {
+  UpdateInfoStepYupSchema,
+  ProjectDraftStepYupSchema,
+  SelectProjectPropertiesStepYupSchema,
+  GreTransferStepYupSchema,
+} from '../../dispose/forms/disposalYupSchema';
+import { Button } from 'components/common/form';
+import { formatDate } from 'utils';
+import styled from 'styled-components';
+import StepErrorSummary from '../../common/components/StepErrorSummary';
+import GenericModal from 'components/common/GenericModal';
+import { GreTransferForm } from '..';
+
+export const GreTransferStepSchema = UpdateInfoStepYupSchema.concat(
+  ProjectDraftStepYupSchema,
+).concat(SelectProjectPropertiesStepYupSchema);
+
+const CenterBoldText = styled.div`
+  text-align: center;
+  font-family: 'BCSans-Bold';
+  margin-bottom: 1.5rem;
+`;
+
+const FlexRight = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row-reverse;
```

Comment:
why row-reverse?

Reply:
I've copied this from the nearly identical implementation for **ERP** to **Transfer within GRE**.  @devinleighsmith may be able to answer this question.

Reply:
basically this provides a flex that grows right -> left instead of left -> right.